### ISSUE TITLE
Code review

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#208]
I'll assign to @jbphet when ready for review.

**Comment by zepumph** [comment]
From dev meeting yesterday, @jbphet will be the one to do the whole code review. I'll rename to reflect that change. Right now I'm shooting for this sometime mid-November, but more development work, self-code review, and likely dev testing will need to be done first.

**Comment by zepumph** [comment]
#210 is done.  This is ready to go, but I'm noticing I accidentally committed a couple of commits to here instead of #210. There also isn't the code-review md in the first comment. I think it would be best to make a new issue.
